src/shared/GlobalTopBanners/GlobalTopBanners.tsx

```diff
@@ -14,13 +14,33 @@ const GlobalTopBanners: React.FC = () => {
   return (
     <SilentNetworkErrorWrapper>
       <div className="[&>*:last-child]:block">
-        <OktaBanners />
-        <RequestInstallBanner />
-        <TrialBanner />
-        <TeamPlanFeedbackBanner />
-        <ProPlanFeedbackBanner />
-        <BundleFeedbackBanner />
-        <TokenlessBanner />
+        <SilentNetworkErrorWrapper>
+          <OktaBanners />
+        </SilentNetworkErrorWrapper>
+
+        <SilentNetworkErrorWrapper>
+          <RequestInstallBanner />
+        </SilentNetworkErrorWrapper>
+
+        <SilentNetworkErrorWrapper>
+          <TrialBanner />
+        </SilentNetworkErrorWrapper>
+
+        <SilentNetworkErrorWrapper>
+          <TeamPlanFeedbackBanner />
+        </SilentNetworkErrorWrapper>
+
+        <SilentNetworkErrorWrapper>
+          <ProPlanFeedbackBanner />
+        </SilentNetworkErrorWrapper>
+
+        <SilentNetworkErrorWrapper>
+          <BundleFeedbackBanner />
+        </SilentNetworkErrorWrapper>
+
+        <SilentNetworkErrorWrapper>
+          <TokenlessBanner />
+        </SilentNetworkErrorWrapper>
```

Comment:
Switching this up, so that we're able to show some banners and not others, for instance if we're on a repo that the user needs to auth with Okta with, before this would just hide all the banners.